File: 기본 문법/9325/9325.py
test_case = int(input())

for i in range(test_case):
    car_price = int(input())
    options = int(input())
    for j in range(options):
        option_num, option_price = map(int, input().split())
        car_price = car_price + option_num * option_price

    print(car_price)

File: 기본 문법/9325/9325.cpp
#include <bits/stdc++.h>
using namespace std;

int main(void) {
    int test_case;
    scanf("%d", &test_case);

    while (test_case--) {
        int car_price;
        scanf("%d", &car_price);

        int options;
        scanf("%d", &options);
        while (options--) {
            int option_num, option_price;
            scanf("%d %d", &option_num, &option_price);

            car_price += option_num * option_price;
        }
        printf("%d\n", car_price);
    }

    return 0;
}